### PR TITLE
Add groups to schema

### PR DESCRIPTION
**Pull request by fijshion**
This is the basics for adding groups to the response when GETing a Host. More features (listing groups, filtering hosts by group, etc.) will be added in future commits.

### CODE REVIEW

File: deploy/ephemeral.yaml [hunk 1]
```diff
@@ -350,6 +355,29 @@ objects:
...
+                  },
+                  "org_id": {
+                    "type": "keyword"
+                  },
+                  "created_on": {
+                    "type": "keyword"
+                  },
+                  "modified_on": {
+                    "type": "keyword"
+                  }
```
Comment (kruai) [inline deploy/ephemeral.yaml:378]: Should these two be `"type": "date_nanos"`?

Comment (fijshion) [reply deploy/ephemeral.yaml]: fixed

**Comment by kruai** [comment]
It looks like the build is failing bc the xjoin pipeline isn't deploying. Isn't that generally caused by xjoin-operator, though?
Otherwise I'd guess it's something in `deploy/ephemeral.yaml`

**Comment by fijshion** [comment]
@kruai tests are passing now

**Review by kruai — APPROVED** [review]

**Pull request merged**